stats/stats.go

```diff
@@ -0,0 +1,82 @@
+// +build linux
+
+package stats
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	pb "github.com/c-fs/cfs/proto"
+	"github.com/google/cadvisor/info/v1"
+	"github.com/google/cadvisor/manager"
+	"github.com/google/cadvisor/storage/memory"
+	"github.com/google/cadvisor/utils/sysfs"
+	"golang.org/x/net/context"
+)
+
+const (
+	// TODO: make container name configurable
+	//
+	// The recommended way to run cfs is putting the process in an
+	// exclusive container, which has its own cgroup in each hierarchy.
+	// It helps cfs to monitor its resource usage.
+	//
+	// It asks users to specify container name instead of detecting
+	// cgroup that has the process automatically because it may meet
+	// strange cases:
+	// 1. cfs process may be in different cpu/memory/etc cgroups
+	// 2. the cgroup that includes cfs may have other processes
+	// So it hopes that user could take care of it.
+	DefaultContainerName = "/cfs"
+
+	statsToCacheNum = 60
+	storageDuration = 2 * time.Minute
+)
+
+var globalServer *server
```

Comment:
> remove the global prefix?

k

> i am also wondering why do we need the global one? can't we create new one every time we need it?

It will maintain some long-time info in the future in this server, e.g., QPS on one disk. It could be some global var or one field in current `globalServer`. Have tried that because that sounds more normal for stats package.